content/influxdb/v2.0/monitor-alert/notification-endpoints/create.md

```diff
@@ -43,4 +44,7 @@ To send notifications about changes in your data, start by creating a notificati
       - [Create a new service](https://support.pagerduty.com/docs/services-and-integrations#section-create-a-new-service), [add an integration for your service](https://support.pagerduty.com/docs/services-and-integrations#section-add-integrations-to-an-existing-service), and then enter the PagerDuty integration key for your new service in the **Routing Key** field.
       - The **Client URL** provides a useful link in your PagerDuty notification. Enter any URL that you'd like to use to investigate issues. This URL is sent as the `client_url` property in the PagerDuty trigger event. By default, the **Client URL** is set to your Monitoring & Alerting History page, and the following included in the PagerDuty trigger event: `"client_url": "https://us-west-2-1.aws.cloud2.influxdata.net/orgs/<your-org-ID>/alert-history”`
 
+      - For Telegram:
+        - Enter a Telegram [**Bot Token**](https://core.telegram.org/bots#generating-an-authorization-token) and [Chat ID](/kapacitor/v1.5/event_handlers/telegram/).
```

Comment:
I think it's weird to reference the Kapa content here....since while it's similar in terms of the feature/function and capability, this doesn't have anything to do with Kapa itself.  So, I think I'd pull this information into the 2.0 docs or perhaps reference the Flux docs where the underlying function and parameters reside? 

https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/contrib/telegram/endpoint/

Reply:
Makes sense. I linked there instead and updated the language from "channel ID" to "chat ID" to match the UI. (Confirmed they're the same thing).